src/crypto/SecretKey.cpp

```diff
@@ -327,16 +328,13 @@ PubKeyUtils::verifySig(PublicKey const& key, Signature const& signature,
 SignatureHint
 PubKeyUtils::getHint(PublicKey const& pk)
```

Comment:
Just have people call SignatureUtils::getHint  directly if you want to move this function there.